apps/www/pages/storage/Storage.tsx

```diff
@@ -113,7 +115,12 @@ function StoragePage() {
                 />
               </div>
               <h4 className="h4">Dependable</h4>
-              <p className="p">Enterprise-level scalability and durability.</p>
+
+              <p className="p">
+                Multiple Protocol Support: <br /> S3, Resumable Uploads and Standard Uploads
```

Comment:
Could `Multiple Protocol Support` be the title instead of `Dependable`?

Reply:
So we don't have multiple line breaks which look weird.

Reply:
yea, thats better